src/org/robockets/steamworks/subsystems/GearIntake.java

```diff
@@ -15,7 +15,7 @@ public void initDefaultCommand() {
 	}
 
 	public boolean isGearInRobot() {
-		return RobotMap.gearInputBreakbeamSensor.get();
+		return false;//RobotMap.gearInputBreakbeamSensor.get();
```

Comment:
Did fixing the port conflict with the encoder fix this issue? Should we undo this?